spec/generator/target_spec.rb

```diff
@@ -20,54 +20,45 @@ module Generator
       end
 
       context 'when visiting a target' do
-        # TODO: Refactor
-        it "should create a new target" do
-          target = double("Target").as_null_object
-          expect(@project).to receive(:new_target).with(target)
+        before :each do
+          @target = double("Target").as_null_object
 
-          @generator.visit_target(target)
+          @native_target = double("Native Target")
+          allow(@project).to receive(:new_target).and_return(@native_target)
         end
 
-        it "should include files" do
-          native_target = double("Native Target")
-          allow(@project).to receive(:new_target).and_return(native_target)
-
-          target = double("Target").as_null_object
+        it "should create a new target" do
+          expect(@project).to receive(:new_target).with(@target)
+          @generator.visit_target(@target)
+        end
 
-          allow(target).to receive(:include_files).and_return(['**/*.swift'])
-          allow(target).to receive(:exclude_files).and_return(nil)
+        it "should include files" do
+          allow(@target).to receive(:include_files).and_return(['**/*.swift'])
+          allow(@target).to receive(:exclude_files).and_return(nil)
           allow(Dir).to receive(:glob).and_return(['File.swift'])
 
-          expect(@root_node).to receive(:create_children_with_path).with('File.swift', native_target)
+          expect(@root_node).to receive(:create_children_with_path).with('File.swift', @native_target)
 
-          @generator.visit_target(target)
+          @generator.visit_target(@target)
         end
 
         it "should exclude files" do
-          native_target = double("Native Target")
-          allow(@project).to receive(:new_target).and_return(native_target)
-
-          target = double("Target").as_null_object
-
-          allow(target).to receive(:include_files).and_return(nil)
-          allow(target).to receive(:exclude_files).and_return(['**/*.swift'])
+          allow(@target).to receive(:include_files).and_return(nil)
+          allow(@target).to receive(:exclude_files).and_return(['**/*.swift'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.